test/e2e/create_test.go

```diff
@@ -126,9 +191,15 @@ func (tc *testContext) waitForWindowsMachines(machineCount int, phase string) er
 			machineStateTimeLimit = time.Minute * 20
 		}
 	}
+
+	listOptions := metav1.ListOptions{LabelSelector: clusterinfo.MachineE2ELabel + "=true"}
+	if windowsLabel {
+		listOptions.LabelSelector += "," + controllers.MachineOSLabel + "=Windows"
+	} else {
+		listOptions.LabelSelector += "," + controllers.MachineOSLabel + "!=Windows"
```

Comment:
I am not sure I completely understand this part. For my own learning --

1. BYOH nodes will not have the `machine.openshift.io/os-id=Windows` label, correct?
2. What sort of machines would be caught with a label selector of `!=Windows`? All non-Windows machines, including  Linux machines?